### PR TITLE
dev-util/drone: New ebuild

### PR DESCRIPTION
**Pull request by mtsmfm**

### CODE REVIEW

File: dev-util/drone/files/droned.init [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#! /bin/sh
```
Comment (ursm) [inline dev-util/drone/files/droned.init]: Hmm... this doesn't seem to be Gentooish so much. Can you rewrite using runscript?

Comment (ursm) [inline dev-util/drone/files/droned.init]: Documentation is here: https://wiki.gentoo.org/wiki/Handbook:X86/Working/Initscripts#Writing_initscripts

Comment (mtsmfm) [inline dev-util/drone/files/droned.init]: I see.
I'll rewrite with runscript :pencil2:

**Comment by ursm** [comment]
Looks good :star2:

**Pull request merged**